glue_jupyter/table/table.vue

```diff
@@ -63,8 +67,13 @@
 </template>
 
 
-<style scoped>
+<style id="glue_table">
```

Comment:
why this change? is this for the hot reloading?

Reply:
It's a small fix as `scoped` doesn't work in the ipyvue context and ideally all style tags should have an id to prevent duplicate style tags and leaking old styles during development.

It's not related to this PR, and shouldn't be here.

Reply:
ok, we can leave it in, but good to know it's not related.